scripts/test-helm.sh

```diff
@@ -98,6 +98,7 @@ pushd "$HELM_DIR" 1> /dev/null
     --set image.repository="$IMAGE_REPO" \
     --set image.tag="$IMAGE_TAG" \
     --set metrics.service.create=true \
+    --wait \
```

Comment:
this is not needed. The pod will never be in ready state because it fails to start due to lack of AWS creds.

AWS creds are added after the helm install and controller starts successfully after that.

Reply:
The tests do pass locally with this. 

I'm not sure I understand what you mean.  The `wait` expects the controller pod to get to a `Ready` state.  If credentials are not valid, then it will eventually fail. But in the normal case where credentials are valid, this is just better than only relying on the 10 sec sleep that is directly below this. 

I might be missing something here :)